TOOLS/matroska.py

```diff
@@ -300,8 +298,8 @@ def parse_elems(l, namespace):
 parse_elems(elements_matroska, 'MATROSKA')
 
 def printf(out, *args):
-    out.write(u' '.join([str(x) for x in args]))
-    out.write(u'\n')
+    out.write(' '.join([str(x) for x in args]))
```

Comment:
If you're editing this line anyway you can also remove the unnecessary list instantiation
```suggestion
    out.write(' '.join(str(x) for x in args))
```